src/html/dom-module-name-or-is.ts

```diff
@@ -49,18 +49,32 @@ class DomModuleNameOrIs extends HtmlRule {
         p.hasTagName('dom-module'), p.OR(p.hasAttr('is'), p.hasAttr('name')));
     const badModules = dom5.queryAll(document.ast, badModule);
     for (const domModule of badModules) {
+      const isFixable =
+          !(dom5.getAttribute(domModule, 'is') !== null &&
+            dom5.getAttribute(domModule, 'name') !== null);
       for (const badAttr of ['is', 'name']) {
         const attr = dom5.getAttribute(domModule, badAttr);
-        if (attr != null) {
+        if (attr !== null) {
+          const sourceRange =
+              document.sourceRangeForAttributeName(domModule, badAttr)!;
```

Comment:
Not your bug but I just noticed it. We should check to see if sourceRange is undefined, and `continue` if so.